### PR TITLE
docs: updated README.md with conda and poetry instrunctions

### LINKED ISSUE #55: Update README.md file with instructions on installing conda and poetry

### PR DESCRIPTION
**Pull request by Rachel0619**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,13 +31,31 @@ Pythoshop complements existing image processing libraries in the Python ecosyste
...
+- Windows: Open the downloaded .exe file and follow the instructions. It's recommended to check the option to "Add Anaconda to my PATH environment variable" for easy use in the command prompt, but be aware this can interfere with other software.
+- macOS or Linux: Open Terminal, navigate to the directory containing the downloaded script, and run it with bash Anaconda3-xxxxxx.sh, following the on-screen instructions.
+3. Verify Installation:
+Open your command line interface (CLI) and type `conda list`. If Anaconda is installed properly, you should see a list of installed packages.
+
+**To install `poetry`, follow the instructions below:**
+
+we recommend installing `Poetry` using their official installer while referring to their official [poetry documentation](https://python-poetry.org/docs/) for detailed installation instructions and support.
+
+Now navigate to the folder that you want to download the pythoshop package.
```
Comment (salva-u) [inline README.md:58]: how can they install?
they need to run poetry install from the root of the Pythoshop folder, can we add this?

**Review by salva-u — APPROVED** [review]
LGTM

**Comment by salva-u** [comment]
Move the current usage stuff into installation

**Review by ella-irene — APPROVED** [review]
Looks good.

**Pull request merged**